samples/wifi/shell/src/wifi_main.c

```diff
@@ -15,4 +16,10 @@ void main(void)
 			       NRF_CLOCK_HFCLK_DIV_1);
 #endif
 	printk("Starting %s with CPU frequency: %d MHz\n", CONFIG_BOARD, SystemCoreClock/MHZ(1));
+
+	/* set clock to be above BSS EXPIRATION value */
+	struct timespec tspec;
+	tspec.tv_sec = 50000;
+	tspec.tv_nsec = 10000;
+	clock_settime(CLOCK_REALTIME, &tspec);
```

Comment:
This looks like a hack, any specific reason? I am debugging a similar bug related to BSS flush from Matter, is it the same?

Reply:
I have root cause and raised https://github.com/nrfconnect/sdk-hostap/pull/53 with proper fix (added your SOB too).